src/ui/primeng/dropdown/src/dropdown.type.html

```diff
@@ -0,0 +1,94 @@
+<p-dropdown
+  [formControl]="formControl"
+  [formlyAttributes]="field"
+  [id]="props?.id ?? null"
+  [scrollHeight]="props?.scrollHeight ?? '200px'"
+  [filter]="props?.filter ?? false"
+  [name]="props?.name ?? null"
+  [style]="props?.style ?? null"
+  [panelStyle]="props?.panelStyle ?? null"
+  [styleClass]="props?.styleClass ?? null"
+  [panelStyleClass]="props?.panelStyleClass ?? null"
+  [readonly]="props?.readonly ?? false"
+  [required]="props?.required ?? false"
+  [editable]="props?.editable ?? false"
+  [appendTo]="props?.appendTo ?? null"
+  [tabindex]="props?.tabindex ?? 0"
+  [placeholder]="props?.placeholder ?? null"
+  [loadingIcon]="props?.loadingIcon ?? null"
+  [filterPlaceholder]="props?.filterPlaceholder ?? null"
+  [filterLocale]="props?.filterLocale ?? locale"
+  [variant]="props?.variant ?? 'outlined'"
+  [inputId]="props?.inputId ?? null"
+  [dataKey]="props?.dataKey ?? null"
+  [filterBy]="props?.filterBy ?? null"
+  [filterFields]="props?.filterFields ?? null"
+  [autofocus]="props?.autofocus ?? false"
+  [resetFilterOnHide]="props?.resetFilterOnHide ?? false"
+  [checkmark]="props?.checkmark ?? false"
+  [dropdownIcon]="props?.dropdownIcon ?? null"
+  [loading]="props?.loading ?? false"
+  [optionLabel]="props?.optionLabel ?? null"
+  [optionValue]="props?.optionValue ?? null"
+  [optionDisabled]="props?.optionDisabled ?? null"
+  [optionGroupLabel]="props?.optionGroupLabel ?? 'label'"
+  [optionGroupChildren]="props?.optionGroupChildren ?? 'items'"
+  [autoDisplayFirst]="props?.autoDisplayFirst ?? true"
+  [group]="props?.group ?? false"
+  [showClear]="props?.showClear ?? false"
+  [emptyFilterMessage]="props?.emptyFilterMessage ?? null"
+  [emptyMessage]="props?.emptyMessage ?? null"
+  [lazy]="props?.lazy ?? false"
+  [virtualScroll]="props?.virtualScroll ?? false"
+  [virtualScrollItemSize]="props?.virtualScrollItemSize ?? null"
+  [virtualScrollOptions]="props?.virtualScrollOptions ?? null"
```

Comment:
Credo che le options del virtual scroll per il lazy possono essere gestite in automatico. In pratica integrare il codice ts qui https://primeng.org/dropdown#lazyvirtualscroll direttamente nel componente.